build.assets/charts/Dockerfile-distroless

```diff
@@ -32,4 +33,4 @@ FROM $BASE_IMAGE
 COPY --from=teleport /opt/staging /
 COPY --from=staging /opt/staging/root /
 COPY --from=staging /opt/staging/status /var/lib/dpkg/status.d
-ENTRYPOINT ["/usr/bin/dumb-init", "/usr/local/bin/teleport", "start", "-c", "/etc/teleport/teleport.yaml"]
+ENTRYPOINT ["/usr/bin/dumb-init", "/usr/local/bin/teleport", "start", "-c", "/etc/teleport/teleport.yaml", "${TELEPORT_FIPS_FLAG}"]
```

Comment:
This isn't going to work because you can't expand ARGs/ENVs in the ENTRYPOINT. This ends up with a literal `${TELEPORT_FIPS_FLAG}` on the command line. This is according to the Dockerfile docs (paragraph at the end of https://docs.docker.com/engine/reference/builder/#exec-form-entrypoint-example) and my local testing.

The only way to do this in general is to turn the arg into an ENV var and use the shell form of ENTRYPOINT. Then the shell will expand the `${TELEPORT_FIPS_FLAG}`. But because we're using distroless, there is no shell for this.

It does no look like there is any way to have a variable form of `ENTRYPOINT` without using the shell form.

Either this will need a new Dockerfile just for FIPS, or we modify teleport to enable FIPS mode if the `TELEPORT_FIPS` environment variable is set so we don't need a command-line flag. However I don't see that we support much in the way of env vars for flags - just one or two, so perhaps that's also not the right way to go.

Reply:
I'll handle this with a separate Dockerfile for now. Sucks, but seems to be the best option right now.